test/unit/controllers/subscription/ConfirmIndividualTypeControllerSpec.scala

```diff
@@ -194,4 +194,14 @@ class ConfirmIndividualTypeControllerSpec extends ControllerSpec with BeforeAndA
     test(result)
   }
 
+  private def assertRadioButtonIsPresent(
```

Comment:
Is this method used?

Reply:
Yes, in this spec only. I moved this from ControllerSpec